gtsam/hybrid/tests/testGaussianMixture.cpp

```diff
@@ -99,7 +79,11 @@ TEST(GaussianMixture, GaussianMixtureModel) {
   double mu0 = 1.0, mu1 = 3.0;
   double sigma = 2.0;
 
-  auto hbn = GaussianMixtureModel(mu0, mu1, sigma, sigma);
+  HybridBayesNet hbn;
```

Comment:
We should add a comment about what's going on here, since before it was pretty obvious we were defining a `GaussianMixtureModel`.

Reply:
Ok